reference-architecture/gce-ansible/playbooks/openshift-setup.yaml

```diff
@@ -9,19 +9,20 @@
       cloud-provider:
       - "gce"
       cloud-config:
-      - "/etc/origin/gce.ini"
+      - "/etc/origin/cloudprovider/gce.conf"
```

Comment:
For now, you'll probably need to update openshift_hostname with the short hostname to ensure everything works.